tests/Unit/Services/Blockchain/NetworkTest.php

```diff
@@ -18,6 +18,9 @@
     expect($subject->alias())->toBe($config['alias']);
     expect($subject->api())->toBe($config['api']);
     expect($subject->explorerTitle())->toBe(config('app.name'));
+    expect($subject->mainnetExplorerUrl())->toBe($config['mainnetExplorerUrl']);
+    expect($subject->testnetExplorerUrl())->toBe($config['testnetExplorerUrl']);
+    expect($subject->polygonExplorerUrl())->toBe($config['polygonExplorerUrl']);
```

Comment:
we can get rid of this as well as it was only necessary to link migration transactions

Reply:
can be another PR

Reply:
:thinking: added this because it was failing tests

Reply:
Ah unless you mean the whole thing since you've merged :oleglala: